package.json

```diff
@@ -8,55 +8,56 @@
   },
   "dependencies": {
     "babel-core": "6.3.21",
-    "babel-plugin-check-es2015-constants": "^6.5.0",
-    "babel-plugin-syntax-async-functions": "^6.3.13",
-    "babel-plugin-transform-es2015-arrow-functions": "^6.5.2",
-    "babel-plugin-transform-es2015-block-scoped-functions": "^6.5.0",
-    "babel-plugin-transform-es2015-block-scoping": "^6.5.0",
-    "babel-plugin-transform-es2015-classes": "^6.5.2",
-    "babel-plugin-transform-es2015-computed-properties": "^6.5.2",
-    "babel-plugin-transform-es2015-destructuring": "^6.5.0",
-    "babel-plugin-transform-es2015-for-of": "^6.5.2",
-    "babel-plugin-transform-es2015-function-name": "^6.5.0",
-    "babel-plugin-transform-es2015-literals": "^6.5.0",
-    "babel-plugin-transform-es2015-modules-commonjs": "^6.5.2",
-    "babel-plugin-transform-es2015-object-super": "^6.5.0",
-    "babel-plugin-transform-es2015-parameters": "^6.5.0",
-    "babel-plugin-transform-es2015-shorthand-properties": "^6.5.0",
-    "babel-plugin-transform-es2015-spread": "^6.5.2",
-    "babel-plugin-transform-es2015-sticky-regex": "^6.5.0",
-    "babel-plugin-transform-es2015-template-literals": "^6.5.2",
-    "babel-plugin-transform-es2015-typeof-symbol": "^6.5.0",
-    "babel-plugin-transform-es2015-unicode-regex": "^6.5.0",
-    "babel-plugin-transform-proto-to-assign": "^6.5.0",
-    "babel-plugin-transform-regenerator": "^6.5.2",
-    "babel-preset-es2015": "^6.5.0",
+    "babel-plugin-check-es2015-constants": "6.5.0",
```

Comment:
I got rid of all the `^` in this file.  This is because different compile versions and different uglify-js versions actually act a little differently, so a lot of interaction tests were failing just because my `npm install` gets different versiosn than whoever wrote these.  In the future we can upgrade these and change test expectations, but for now I think this is fine.
(One example of different behavior: in some versions `const x, y, z;` gets compiled to `var x, y, z;` and in some `var x; var y; var z;`.  Another example is `var hi = [1,2,3,4]; someFunc(hi);` will get uglified in some versions to `var a = [1,2,3,4]; b(a)` and in others it'll get inlined to `b([1,2,3,4])` - so version changes would require a lot of tst expectation changes everywhere.)